web/EXAMPLES.md

```diff
@@ -24,15 +24,20 @@ For each third party asset request on the site, take the hostname of a request,
         - If a rule match is found, then for the first matching rule:
 
             a. If the rule has `action: ignore`, do not block[^2]
+	    
+	      b. If there are rule `options`, do any of the domains and request types match this request?[^1]
 
-            b. If there are rule `exceptions`, do any of the domains and request types match this request?[^1]
+	      - No: Continue to next rule
```

Comment:
We are saying above "and only one rule can match per request". In this case we consider that rule did not match, right?

Reply:
Yes, if we didn't match domains/types then it is not a match.